providers/Sentry.js

```diff
@@ -8,7 +8,11 @@ class SentryProvider extends ServiceProvider {
     const Config = this.app.use('Adonis/Src/Config')
     this.app.singleton('Sentry', () => {
       const sentryDns = Config.get('sentry.dns')
-      Sentry.init({ dsn: sentryDns })
+      const environment = Config.get('sentry.environment')
+      Sentry.init({ 
+        dsn: sentryDns,
+        environment
```

Comment:
This should be `environment : environment ` or
```
Sentry.init({ 
        dsn: Config.get('sentry.dns'),
        environment: Config.get('sentry.environment')
})
```

Reply:
if you have the same name on key:value you can use only key.
https://eslint.org/docs/rules/object-shorthand